app/javascript/components/game/Game.js

```diff
@@ -1,97 +1,98 @@
+    });
+    localStorage.setItem("gameId", form.id);
+  };
 
-        }
-
-        playGame = (output) => {  
-           
-          if (output&&output.message){
-          this.setState({
-              messages: output
-            });
-        }
-      }
-        resumeGame = (game) => { 
-          this.setState({
-              messages: game.data.message,
-              id: game.data.id
-            });
-            localStorage.setItem('gameId', game.data.id)
-      }
-
-    render(){
-        const { messages } = this.state
-        const cells = [];
-        for (let index = 0; index < 9; index++) { 
-          cells.push( <Cell 
-            key={index} 
-            id={this.state.id} 
-            playGame={this.playGame} 
-            position={index} 
-            message={messages.message && messages.message.board[index]}
-            endGame={messages.message && messages.message.win || messages.message && messages.message.draw }
-            />);
-        }
-        return(
-            <>
-            
-                <div className='game_left'>
-                  <div className="container">
-
-                    <div className="row">
-                      <div className="col-md-3">
-                        <GameForm addGame={this.addGame}/>
-                      </div>
-                      
-                      <div className="col-md-6">
-                          <h3>Current Game ID: <span className="badge badge-success">{this.state.id}</span></h3>
-                          <h1>Tic-Tac-Toe</h1>
-                          <ul id="game">
-                            {cells}
-                          </ul>
+  playGame = (output) => {
+    if (output && output.message) {
+      this.setState({
+        messages: output,
+      });
+    }
+  };
+  resumeGame = (game) => {
+    this.setState({
+      messages: { message: game.data.message },
+      id: game.data.id,
+    });
+    localStorage.setItem("gameId", game.data.id);
+  };
 
-                        <div id="game-messages">
-                            <span className="game-over">{messages.message && messages.message.win}</span>
-                            <span className="game-over">{messages.message && messages.message.draw}</span>
-                        </div>
-                      </div>
-                      <div className="col-md-3">
-                        <ResumeGame resumeGame = {this.resumeGame}/>
-                      </div>
-                  </div>
+  render() {
+    const { messages } = this.state;
+    const cells = [];
+    for (let index = 0; index < 9; index++) {
+      cells.push(
+        <Cell
+          key={index}
+          id={this.state.id}
+          playGame={this.playGame}
+          position={index}
+          message={messages.message && messages.message.board[index]}
+          endGame={
+            (messages.message && messages.message.win) ||
+            (messages.message && messages.message.draw)
```

Comment:
We are checking `messages.message` in multiple spots in this file. Is there a way to do it only once?